opm/autodiff/StandardWellsDense.hpp

```diff
@@ -116,7 +120,8 @@ enum WellVariablePositions {
                       const std::vector<double>& depth_arg,
                       const std::vector<double>& pv_arg,
                       const RateConverterType* rate_converter,
-                      long int global_nc);
+                      long int global_nc,
+                      const auto& grid);
```

Comment:
@totto82 This change seems to produce compile errors with gcc 4.8. See the [travis run](https://travis-ci.org/OPM/opm-simulators/builds/247462206?utm_source=github_status&utm_medium=notification) for #1189:
```
In file included from /home/travis/build/OPM/opm-simulators/opm/autodiff/BlackoilModelEbos.hpp:31:0,

                 from /home/travis/build/OPM/opm-simulators/opm/autodiff/SimulatorFullyImplicitBlackoilEbos.hpp:27,

                 from /home/travis/build/OPM/opm-simulators/examples/flow_ebos.cpp:33:

/home/travis/build/OPM/opm-simulators/opm/autodiff/StandardWellsDense.hpp:125:35: error: parameter declared ‘auto’

                       const auto& grid);

                                   ^
``